tests/test_usage.py

```
@@ -651,6 +651,60 @@ def test_Usage_Error_WHEN_the_help_command_is_called(self):
+						context.debugBlob(theOutputtxt)
+						# now test it
+						try:
+							if isinstance(theOutputtxt, bytes):
+								theOutputtxt = theOutputtxt.decode('utf8')
+						except UnicodeDecodeError:
+							theOutputtxt = str(repr(bytes(theOutputtxt)))
+						# or simply:
+						self.assertIsNotNone(theOutputtxt)
+						self.assertIn(str("""usage:"""), str(theOutputtxt))
+						if (str("""usage:""") in str(theOutputtxt)):
+							theResult = True or theResult
```

Ruff

Use `True` instead of `True or ...`

Replace with `True`

(SIM222)